data/json/vehicle_parts.json

```diff
@@ -359,6 +359,24 @@
         ]
     },{
         "type" : "vehicle_part",
+        "id" : "folding_wooden_frame",
+        "name" : "foldable wooden frame",
+        "symbol" : "h",
+        "color" : "brown",
+        "broken_symbol" : "#",
+        "broken_color" : "brown",
+        "durability" : 50,
+        "folded_volume": 15,
+        "difficulty" : 1,
+        "item" : "foldwoodframe",
+        "location" : "structure",
+        "flags" : ["MOUNTABLE", "FOLDABLE", "INITIAL_PART"],
```

Comment:
Lacks NAILABLE, meaning you can't really use it without a non-wood folding frame.